src/tools/sdktool/addsfdkbuildengineoperation.h

```diff
@@ -59,6 +59,7 @@ class AddSfdkBuildEngineOperation : public Operation
            const QUrl &vmUri,
            const QDateTime &creationTime,
            bool autodetected,
+           const QString &sharedInstallPath,
```

Comment:
Nothing wrong with this, but now RmSfdkBuildEngineOperation::test() won't compile anymore, as it doesn't have this parameter

Reply:
oops, fixed.